Cargo.toml

```diff
@@ -13,21 +14,25 @@ path = "./src/main.rs"
 anyhow = "1.0.86"
 axum = "0.7.5"
 clap = { version = "4.5.16", features = ["derive"] }
-holochain = { version = "0.4.0-dev.22", features = ["chc", "test_utils"] }
```

Comment:
Wind Tunnel is still running 0.4.0, and ideally we'd do some testing against that. 

What was the reason for the CHC service depending on `holochain`? Can we use this updated service with Holochain 0.4.0?

Reply:
In general, version bumps belong on a different PR to feature work. Why was this done at the same time?